examples/gno.land/p/sys/antisquatting/auction.gno

```diff
@@ -0,0 +1,393 @@
+package antisquatting
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"std"
+	"strconv"
+	"time"
+	
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/ufmt"
+)
+
+// DefaultAuctionManager implements AuctionManager with commit-reveal auctions
+type DefaultAuctionManager struct {
+	auctions *avl.Tree // name -> *Auction
+	deposits *avl.Tree // bidder_name -> int64 (total deposits)
```

Comment:
`// bidder_name -> int64 (total deposits)`
When I first read the comment I thought it was like
deposits[address] -> deposits
but It is actually {{address}} + "_" + {{auctioned namespace}}

you can close this if you consider it clear enough :)